assets/prometheus-k8s/rules.yaml

```diff
@@ -334,6 +334,18 @@ spec:
     - expr: sum(rate(container_cpu_usage_seconds_total{container_name!="POD",container_name!="",pod_name!=""}[5m]))
         / sum(machine_cpu_cores)
       record: cluster:container_cpu_usage:ratio
+    - expr: sum({__name__="machine_cpu_cores"})
+      record: cluster_cpu_cores
```

Comment:
The best practice is to use `:` to show this is a recording rule and put the result and used aggregation into perspective. So this would be `cluster:machine_cpu_cores:sum`. You can look at examples of the other recording rules defined here.